app/src/test/java/ch/epfl/sdp/blindly/UserCacheUnitTest.kt

```diff
@@ -13,25 +13,28 @@ class UserCacheUnitTest {
         private const val location = "EPFL, Ecublens"
         private const val birthday = "01.01.2001"
         private const val gender = "Woman"
-        private val  sexual_orientations =  listOf("Asexual")
+        private val sexual_orientations = listOf("Asexual")
```

Comment:
Unluckily you modified a line I wrote incorrectly, could you at the same time refactor the sexual_orientations to sexualOrientations.... (please) ? x)

Reply:
```suggestion
        private val sexualOrientations = listOf("Asexual")
```

Reply:
Sorry I completely forgot that I could propose a commit to make this kind of changes :'(
But thanks :)

Reply:
Yeah but that was a bad idea xD